scss/detail.scss

```diff
@@ -0,0 +1,46 @@
+@import 'colors';
+* {font-family: Roboto};
+
+.book-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 125px;
+  margin-left: 10px;
+}
+
+.book-genre {
+  height: 30px;
+  width: 100px;
+  color: $grey;
+  font-size: 25px;
+  font-weight: 700;
+  line-height: 30px;
+}
+
+.book-title {
+  height: 40px;
+  width: 210px;
+  color: $black;
+  font-size: 30px;
+  font-weight: 700;
+  line-height: 40px;
+  margin: 10px 0 40px 10px;
+}
+
+.default-black-text {
+  color: $black;
+  height: 25px;
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 25px;
+  margin-right: 5px;
+}
+
+.default-grey-text {
```

Comment:
I have a suggest for default class

`.default-text {
  color: $grey;
  font-size: 20px;
  line-height: 25px;
}`

and for black-text 

`.category {
  color: $black;
  font-weight: 700;
  margin-right: 5px;
}`

And you can use  `<h3 class="default-text category">Año de publicación:</h3>`